crates/storage/store_db/rocksdb.rs

```diff
@@ -740,9 +743,17 @@ impl StoreEngine for Store {
             }
 
             for (code_hash, code) in update_batch.code_updates {
-                let code_key = code_hash.as_bytes();
-                let code_value = AccountCodeRLP::from(code).bytes().clone();
-                batch.put_cf(&cf_codes, code_key, code_value);
+                let mut buf =
+                    Vec::with_capacity(6 + code.bytecode.len() + 2 * code.jump_targets.len());
+                code.bytecode.encode(&mut buf);
+                unsafe {
+                    std::slice::from_raw_parts(
+                        code.jump_targets.as_ptr().cast(),
+                        2 * code.jump_targets.len(),
+                    )
+                }
+                .encode(&mut buf);
```

Comment:
No real need for unsafe. Just serialize the correct size header and iterate appending the `to_le_bytes`.

Reply:
```suggestion
                ethrex_rlp::encode::encode_length(2 * code.jump_targets.len(), &mut buf);
                buf.extend(code.jump_targets.into_iter().flat_map(|t| t.to_le_bytes()));
```